processing/src/main/java/io/druid/segment/StringDimensionMergerV9.java

```diff
@@ -294,6 +295,7 @@ public void writeIndexes(List<IntBuffer> segmentRowNumConversions, Closer closer
     try (
         Closeable toCloseEncodedValueWriter = encodedValueWriter;
         Closeable toCloseBitmapWriter = bitmapWriter;
+        Closeable toCloseDictionaryMergeIterator = dictionaryMergeIterator;
```

Comment:
Ah okay. Can you please add a comment about this, so future reviewers won't get confused like I did. Other than that it looks good to me.